Marlin/src/lcd/menu/menu.cpp

```diff
@@ -441,12 +441,16 @@ void scroll_screen(const uint8_t limit, const bool is_menu) {
     const bool saved = settings.save();
     #if HAS_BUZZER
       ui.completion_feedback(saved);
+    #else
+      UNUSED(saved);
     #endif
   }
   void lcd_load_settings() {
     const bool loaded = settings.load();
     #if HAS_BUZZER
       ui.completion_feedback(loaded);
+    #else
+      UNUSED(loaded);
     #endif
```

Comment:
An empty `completion_feedback` method in the case of `! HAS_BUZZER` would clean some of this up.

Reply:
This works only if `buzz` works.
At the moment there are difficulties with LCD, therefore my PR.